site/resources.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: Resources
+description: Sonobuoy Resources
+id: resources
+---
+
+<div class="sub-page">
+    <h2>Here you will find external resources about Sonobuoy, such as videos, podcasts, and community articles.</h2>
+    <div class="grid three">
+        <div class="col">
+            <div class="embed-responsive embed-responsive-16by9">
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/1e6SAZfkqUk"></iframe>
+            </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/1e6SAZfkqUk">A first look at Sonobuoy</a></p>
+            </div>
+        </div>
+        <div class="col">
+            <div class="embed-responsive embed-responsive-16by9">
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/k-P4hXdruRs"></iframe>
+            </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/k-P4hXdruRs">Overview of Sonobuoy</a></p>
```

Comment:
@amymanion this shouldn't link to the embed url but should link to the video link on youtube?

Reply:
Fixed this.